src/InitOMPasses.hpp

```diff
@@ -75,5 +79,40 @@ void initOMPasses(int optLevel) {
   mlir::registerPass([]() -> std::unique_ptr<mlir::Pass> {
     return createLowerKrnlShapePass();
   });
+
+#ifdef __NNPA__
+  mlir::registerPass([]() -> std::unique_ptr<mlir::Pass> {
+    return mlir::createONNXToZHighPass();
```

Comment:
I think we should not place the `mlir` namespace with our conversion passes.  Suggest to place those declaration in namespace `onnx_mlir` ?

Reply:
Removed `mlir::`.